workflows/alevin-quant/run-alevin-fry.nf

```diff
@@ -157,5 +166,5 @@ workflow{
   // collate RAD files 
   collate_fry(generate_permit.out)
   // create gene x cell matrix
-  quant_fry(collate_fry.out, params.t2g_path)
+  quant_fry(collate_fry.out, params.t2g_3col_path)
```

Comment:
I'm confused here, which I think ends up related to the question at the top: `params.t2g_3col_path` is not always a 3 column file, correct? If so, maybe we should just keep the name as `t2g_path` for somewhat less confusion (since that is the ambiguous version, to me).